codalab/rest/util.py

```diff
@@ -95,6 +98,25 @@ def wrapper(*args, **kwargs):
     return wrapper
 
 
+# For non-REST services, should call with client=CodaLabManager
+@local_bundle_client_compatible
+def notify_admin(local, request, message):
+    if 'admin_email' in local.config['server']:
```

Comment:
we already have an 'admin-email' field, should be consolidated with 'server/admin_email'

I'd probably go with the latter, but we have to make sure all deployments / documentation is updated accordingly.